docs/source/techniques/ISISPowder-Polaris-v1.rst

```diff
@@ -176,14 +176,25 @@ from focused POLARIS data, with a view performing further total scattering analy
 With no merging criteria specified, *merge_banks=False* a PDF will be generated for each bank within
 the focused_workspace.
 
+This function requires the run_number you which to analyse. The focused file for this run number must
```

Comment:
Nit: presumably should be "run_number you **wish** to analyse"?

Reply:
Yep good spot